lockbox2/programs/liquidity_lockbox/src/lib.rs

```diff
@@ -568,14 +564,15 @@ pub struct WithdrawLiquidityForTokens<'info> {
   pub bridged_token_mint: Box<Account<'info, Mint>>,
   #[account(mut,
     constraint = bridged_token_account.mint == lockbox.bridged_token_mint,
-    constraint = lockbox.bridged_token_mint == bridged_token_mint.key(),
+    constraint = bridged_token_account.mint == bridged_token_mint.key(),
     constraint = signer.key == &bridged_token_account.owner,
   )]
   pub bridged_token_account: Box<Account<'info, TokenAccount>>,
```

Comment:
Switching context of verified accounts to correspond to actual declared accounts.